### DISCUSSION TITLE
Works with large PS2 games up to 4gb or 8gb

### DISCUSSION THREAD

**Discussion by Wesleyjor6** [btimofeev/UniPatcher#47] [Q&A]
Hello I was wondering if the app supports PS2 image with .iso format?

**Comment by btimofeev** [comment]
Do you have any errors while patching such files?  I have never tested on such large files.  Only the patch format is important for the application.  But the application first copies all the files that you select to its internal folder, applies a patch there, then copies it to the output file.  So with large files, there may be a problem with free space.  Or it will take a long time to apply the patch.

**Reply by Wesleyjor6** [reply]
I was trying to apply a patch to the game the shadow of the Colossus I used the right patch in the right game and it was giving an error could you add support for PS2 games? I did in format PS2 games Of cd and work the game had about 700 MB good and cue already in the shadow of the Colossus which are 2 gb was not it did all process more even so it gave error and the patch was running because the Pc worked more Not everyone has a PC. Could you add this support with PS2 games? And one more thing my phone has 32 GB of internal memory

**Reply by btimofeev** [reply]
Only the format of the patch matters. These formats are supported: IPS, IPS32, UPS, BPS, APS (GBA), APS (N64), PPF, DPS, EBP and XDelta3. 

What patch are you using? Give a link to it. Please also write what error the application displays.

**Reply by Wesleyjor6** [reply]
https://www.retro-jogos.com/02/11/2020/ps2-shadow-of-the-colossus/
Will was simply saying that it was possible to apply the patch after he did the procedure, it took until the ISO size reached a message From unknown erro

**Comment by Wesleyjor6** [comment]
![Screenshot_2021-01-18-13-35-53](https://user-images.githubusercontent.com/50889997/104941864-65073300-5992-11eb-8f12-c9e0283dee84.png)

 this is the error that appears every time when I patch the game

**Reply by btimofeev** [reply]
Sorry, but most likely I'm not able to fix this problem at this time.  I am using an external xdelta patching library which gives your error.  But now I don't have time to figure out how it works and why this error occurs.
